src/sponge/pad.rs

```diff
@@ -35,6 +35,26 @@ where
     words
 }
 
+// Padding function for a singular scalar input 
+// to provide two outputs from a Poseidon hash
+pub fn pad_fixed_hash <T>(
+    capacity: T,
+    message: T,
+    pad_value: T,
+) -> Vec<T>
+where
+    T: Clone,
+{
+    // For a constant length hash, we always use a slice of width 5.
+    let width = 5;
+    let zero = pad_value;
+    let mut words = vec![zero; width];
+
+    words[0] = capacity;
+    words[1] = message;
+    words
+}
+
```

Comment:
This pad should not be inside the sponge module. 
Please move it under the same module where the fixed hash file lives.